<!DOCTYPE html>
<html>
<head>
    <title>Core Bioinformatics Induction</title>
    <!-- Bootstrap CSS -->
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
    <link rel="stylesheet" href="https://bootswatch.com/5/flatly/bootstrap.min.css">
    <style>
        <!--
        h1,h2,h3,h4 { color: #343f47; }
        .qigreen { color: #009d80; }
        .qilime {  color: #bed230; }
        .qidark {  color: #343f47; }
        -->
    </style>
</head>
<body>
    <div class="container">
        <h1 class="mt-5 qigreen">Core Bioinformatics Induction</h1>

	<h2 class="mt-5 qilime">Video introductions</h2>
        <div class="list-group mt-4">
            <a href="https://www.youtube.com/watch?v=TPnv5ikhSmk" class="list-group-item list-group-item-action" target="_blank">
		IRIDA, the short reads archiving platform</a>
            <a href="https://www.youtube.com/watch?v=o0egg4eDH6M&t=23s" class="list-group-item list-group-item-action" target="_blank">
		QIB Galaxy, self-service platforms for bioinformatics</a>
            <a href="https://www.youtube.com/watch?v=FMjBXtVzbHo&t=101s" class="list-group-item list-group-item-action" target="_blank">
		QIB Cloud, how to use a virtual machine</a>
        </div>
    </div>


    <div  class="container my-5">
    <h2 class="mt-5 qilime">General information</h2>
    <p>See: <a href="https://quadram-institute.atlassian.net/servicedesk/customer/portal/3/article/4423813">documentation pages (login required)</a></p>
    <h3>Who should you contact?</h3>
    <p>As you may remember from your Computing Orientation Seminar, there are two teams ("Core Computing" and "Research Computing" [RC]) 
        responsible for your digital life at the NBI. 
        We are neither of those: we help with the bioinformatics tasks (software, scientific data) produced and/or analysed at the Quadram. 
        There are grey areas, like the installation of scientific software in the HPC 
        (if you ask us, we can install it first and then ask RC to make it "official" later). 
        But we don't have any control over, for instance, your password, e-mail, or hardware in general.
    </p>
    <p>You can still, however, ask us about those using the Help Centre interface, and we will find an appropriate person to help you.</p>
    <p>There is a dedicated support team for Food Microbiome and Health (FMH), which includes Andrea Telatin and Sumeet Tiwari.</p>
    <p>There is documentation regarding the previous ISPs, Gut Microbes and Health and Food Innovation and Health.</p>
    <p>You should send help requests to our shared email support@quadram-institute.atlassian.net</p>
    
    <h3>Bioinformatics resources are QIB</h3>
    <p>The core bioinformatics team maintains a number of resources to help you get your work done efficiently and effectively.</p>
    <ul>
        <li>Data storage documentation</li>
        <li>Galaxy Documentation</li>
        <li>IRIDA Documentation</li>
        <li>OMERO Documentation</li>
        <li>QIB Cloud Documentation</li>
    </ul>
    <p>Furthermore, in case you are more involved in computational work, the bioinformatics team is quite active in the Data Science group at the QIB.</p>
    
    <h4>Data storage facilities</h4>
    <p>The core bioinformatics team oversees the setting up and maintaining systems for storing and organizing sequencing and related data in a secure and accessible manner.</p>
    
    <ul>
    <li><p><strong>NBI HPC</strong>
    The HPC facilities provide Institutes with the significant ‘compute’ power required for the processing and analysis of genomic/proteomic data (bioinformatics),
	    as well as modelling, simulations and other areas of study.</p>
    <p>HPC stands for High-Performance Computing. It refers to using powerful computing systems, 
	    often consisting of multiple interconnected computers or servers, to perform complex and computationally demanding tasks. 
	    HPC systems are designed to deliver high computational performance, large storage capacity, and high-speed data transfer capabilities.</p>
    <p>The HPC at NBI is managed by Research computing. Core Bioinformatics at QIB are users of the system, just like you, 
	    we can assist and liaise with Research computing on any issues.</p></li>

    <li><p><strong>QIB-Cloud</strong>
    In the QIB Cloud, OpenStack handles provisioning, and a web interface based on Bryn interface allows users to control their resources. It is modelled after CLIMB-BIG-DATA.</p></li>
    
    <li> <p><strong>OMERO</strong>
   OMERO stands for Open Microscopy Environment Remote Objects. It is an open-source software platform designed for managing, sharing, and analyzing biological microscopy data. The primary goal of OMERO is to facilitate the storage, organization, and collaboration of large sets of microscope images and related metadata.</p></li>
    
    <li><p><strong>IRIDA</strong>
    We use IRIDA as the primary storage for sequencing reads. Notice that you must upload your (non-human) data to IRIDA if the Quadram have any involvement in the project.</p></li>

    <li><p><strong>Galaxy:</strong>
    Galaxy is the front-end from where we submit all our jobs to the cluster. It contains most sequence-based bioinformatics software, and you can create or use existing workflows.</p>
    </li>
     </ul>
    <h4>To get started</h4>
    <p>Right now, you can use your Quadram e-mail+password to automatically create an account on Galaxy. You can also contact us to create an IRIDA account for you.</p>

    <h4>NBI and Quadram credentials</h4>
    <p>If you are new to the Quadram, you may have noticed that the Computing Services folks gave you both a short username and a long e-mail. To connect to the HPC or to your local machine, you need to enter the short version, while other services request the e-mail.</p>
</div>
 

 
    <!-- Optional JavaScript -->
    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
</body>
</html>
